SpringProject/src/test/java/com/gradproject2019/PasswordServiceTest.java

```diff
@@ -0,0 +1,43 @@
+package com.gradproject2019;
+
+import com.gradproject2019.users.service.PasswordService;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PasswordServiceTest {
+
+    @InjectMocks
```

Comment:
No need to use Mockito here. You are not using mocks anywhere in this test.

Reply:
Removing this results in a null pointer exception?

Reply:
Your Password service doesn't have any dependency, it's a simple util class, you can create it via new, or in general make it static class with utility methods in it. It doesn't have any business logic in it.

Reply:
In the future this will become part of a new Authorisation Controller, Service, Repository sequence, which is why it's called service and has the `@Service` tag. does that mean we would have to have the mockito stuff?

Reply:
"In the future" is a very abstract thing (it could be tomorrow, but it could be in 2 months time). If you do not need it now, then you have to keep it minimal.